save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -374,7 +378,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
 
                         button(type = ButtonType.button, classes = "btn") {
                             fontAwesomeIcon(icon = faTimesCircle)
-                            attrs.hidden = state.isRoleViewer == true || (state.isEditDisabled ?: true)
+                            attrs.hidden = state.selfRole.isLowerThan(Role.ADMIN) || (state.isEditDisabled)
```

Comment:
Should we create methods like `hasReadAccess`, `hasWriteAccess` on the frontend too? Looks like there is a lot of comparisons with specific roles

Reply:
Well, not sure if we want to enlarge `ProjectView` or `OrganizationView`

Reply:
Done